app/models/user.rb

```diff
@@ -87,6 +87,10 @@ def supervisor
       where(name: 'supervisor')
     end
 
+    def coach
```

Comment:
This could be a good time to refactor all these similar methods (except the `admin` one) with some meta programming. Ping me if you need any pointers.

Reply:
ok, this one we have to study a little bit but we will add the refactor soon. :dancer:

Reply:
👍